python/lsst/obs/pfs/pfsMapper.py

```diff
@@ -216,6 +341,18 @@ def std_raw(self, item, dataId):
         if dataVersion is not None and dataVersion <= 0x0070:
             self._shiftAmpPixels(exp)
 
+        if np.nanmean(exp.variance.array) == 0:   # variance isn't set
+            gain = exp.getDetector()[0].getGain()
+
+            if exp.getFilterLabel().physicalLabel == 'n':
+                gain *= md["W_H4GAIN"]
+                var = exp.image.array/gain
+                var *= 2                        # CDS
```

Comment:
But it's possible that "You are processing a single frame" and CDS wasn't applied.

Reply:
I think that'd be in `std_rawb`, not here.